src/app/app.module.ts

```diff
@@ -5,16 +5,51 @@ import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { HttpClientModule } from "@angular/common/http";
+import { HomeComponent } from './pages/home/home.component';
+import { CreateCategoryComponent } from './pages/category/create-category/create-category.component';
+
+import { MatListModule } from '@angular/material/list';
+import { MatButtonModule } from '@angular/material/button';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatIconModule } from '@angular/material/icon';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatInputModule } from '@angular/material/input';
+import { EditCategoryComponent } from './pages/category/edit-category/edit-category.component';
+import { CategoryDetailComponent } from './pages/category/category-detail/category-detail.component';
+import { MatOptionModule } from '@angular/material/core';
+import { MatSelectModule } from '@angular/material/select';
+import { FormsModule } from '@angular/forms';
+import { MatTableModule } from '@angular/material/table';
+import { CreateTaskComponent } from './pages/task/create-task/create-task.component';
+import { EditTaskComponent } from './pages/task/edit-task/edit-task.component';
+import { MatCheckboxModule } from '@angular/material/checkbox';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    HomeComponent,
+    CreateCategoryComponent,
```

Comment:
you  should use SCAM or Standalone because very soon your app.module.ts will become very big